### PR TITLE
[New] Converted test cases for approvals components from enzyme to rtl

### PR DESCRIPTION
**Pull request by gitdivyanshu**

### CODE REVIEW

File: app/javascript/components/approvals/components/detail/component.spec.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+  // it("render a DisplayData", () => {
+  //   expect(component.find(DisplayData)).to.have.lengthOf(4);
+  // });
+
+  it("render the correct approvals label", () => {
+    expect(screen.getAllByText(CASE_PLAN)).toBeTruthy();
+  });
+
+  it('should accept valid props', () => {
```
Comment (jtoliver-quoin) [inline app/javascript/components/approvals/components/detail/component.spec.js:48]: remove this test

File: app/javascript/components/approvals/components/detail/component.spec.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  it("render ApprovalDetail component", () => {
+    const element = screen.getByText("approvals.response_for_label");
+    expect(element).toBeInTheDocument();
+  });
+
+  // it("render a Grid", () => {
+  //     expect(screen.getAllByRole('div', { className: 'MuiGrid-root' })).toBeDefined();
+  // });
+
+  // it("render a DisplayData", () => {
```
Comment (jtoliver-quoin) [inline app/javascript/components/approvals/components/detail/component.spec.js:40]: This should be testable

File: app/javascript/components/approvals/components/detail/component.spec.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  beforeEach(() => {
+    mountedComponent(<ApprovalDetail {...props} />, initialState);
+  });
+
+  it("render ApprovalDetail component", () => {
+    const element = screen.getByText("approvals.response_for_label");
+    expect(element).toBeInTheDocument();
+  });
+
+  // it("render a Grid", () => {
```
Comment (jtoliver-quoin) [inline app/javascript/components/approvals/components/detail/component.spec.js:36]: Remove this test

File: app/javascript/components/approvals/components/summary/component.spec.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+  // it("render a Grid", () => {
+  //   expect(component.find(Grid)).to.have.lengthOf(3);
+  // });
+
+  // it("render a Chip", () => {
+  //   expect(component.find(Chip)).to.have.lengthOf(1);
+  // });
+
+  it('should accept valid props', () => {
```
Comment (jtoliver-quoin) [inline app/javascript/components/approvals/components/summary/component.spec.js:47]: Remove this test

File: app/javascript/components/approvals/components/panel/component.spec.js [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+import ApprovalPanel from "./component";
+import { fromJS } from "immutable";
+import { Accordion, AccordionDetails, AccordionSummary } from "@material-ui/core";
+import ApprovalSummary from "../summary";
+import ApprovalDetail from "../detail";
+import { CASE_PLAN, STATUS_APPROVED } from "../../constants";
+describe("<ApprovalPanel /> - Component", () => {
+  const props = {
+    approvalSubform: fromJS({
+      approval_date: "2020-01-01",
```
Comment (jtoliver-quoin) [inline app/javascript/components/approvals/components/panel/component.spec.js:11]: Should test that text from these props are rendered to the screen

File: app/javascript/components/approvals/components/panel/component.spec.js [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+  // it("render a ApprovalSummary", () => {
+  //   expect(component.find(ApprovalSummary)).to.have.lengthOf(1);
+  // });
+
+  // it("render a ApprovalSummary", () => {
+  //   expect(component.find(ApprovalDetail)).to.have.lengthOf(1);
+  // });
+
+  it('should accept valid props', () => {
```
Comment (jtoliver-quoin) [inline app/javascript/components/approvals/components/panel/component.spec.js:41]: Remove this test

**Review by jtoliver-quoin — APPROVED** [review]

File: app/javascript/components/approvals/components/detail/component.spec.js (hunk 1 above)
Comment (gitdivyanshu) [reply app/javascript/components/approvals/components/detail/component.spec.js]: done

File: app/javascript/components/approvals/components/detail/component.spec.js (hunk 2 above)
Comment (gitdivyanshu) [reply app/javascript/components/approvals/components/detail/component.spec.js]: done

File: app/javascript/components/approvals/components/detail/component.spec.js (hunk 3 above)
Comment (gitdivyanshu) [reply app/javascript/components/approvals/components/detail/component.spec.js]: done

File: app/javascript/components/approvals/components/summary/component.spec.js (hunk 4 above)
Comment (gitdivyanshu) [reply app/javascript/components/approvals/components/summary/component.spec.js]: done

File: app/javascript/components/approvals/components/panel/component.spec.js (hunk 5 above)
Comment (gitdivyanshu) [reply app/javascript/components/approvals/components/panel/component.spec.js]: done

File: app/javascript/components/approvals/components/panel/component.spec.js (hunk 6 above)
Comment (gitdivyanshu) [reply app/javascript/components/approvals/components/panel/component.spec.js]: done

**Pull request merged**